Rubberduck.Refactorings/ImplementInterface/ImplementInterfaceRefactoring.cs

```diff
@@ -125,126 +129,68 @@ private void AddItems(List<Declaration> missingMembers, IModuleRewriter rewriter
         }
 
         private string GetInterfaceMember(Declaration member, string interfaceName)
-        {
+        {          
+            var template = string.Join(Environment.NewLine, "Private {0}{1} {2}{3}", MemberBody, "End {0}", string.Empty);
+            var signature = $"{interfaceName}_{member.IdentifierName}({string.Join(", ", GetParameters(member))})";
+            var asType = $" As {member.AsTypeName}";
+
             switch (member.DeclarationType)
             {
                 case DeclarationType.Procedure:
-                    return SubStmt(member, interfaceName);
-
+                    return string.Format(template, "Sub", string.Empty, signature, string.Empty);
                 case DeclarationType.Function:
-                    return FunctionStmt(member, interfaceName);
-
+                    return string.Format(template, "Function", string.Empty, signature, asType);
                 case DeclarationType.PropertyGet:
-                    return PropertyGetStmt(member, interfaceName);
-
+                    return string.Format(template, "Property", " Get", signature, asType);
                 case DeclarationType.PropertyLet:
-                    return PropertyLetStmt(member, interfaceName);
-
+                    return string.Format(template, "Property", " Let", signature, string.Empty);
```

Comment:
In this whole switch block, we should work with the relevant tokens from the Parser to avoid accidental typos.

Also the template can be changed a bit to have redundant space. double spaces can then be dropped with `string.ReplaceAll("  ", " ");`

Reply:
Agreed and fixed.